### DISCUSSION TITLE
Teradata Connection Advanced Settings missing

### DISCUSSION THREAD

**Discussion by Crudelis-rgb** [dbeaver/dbeaver#17267] [Q&A]
Dear DBeavers!

I was looking into connecting to my Teradata instance with DBeaver.
It was announced that DBeaver does support a hierarchical view in the Navigator Windows (see https://twitter.com/dbeaver_news/status/1506245166167961607). Now, there should be a corresponding option in the Connection Settings under "Advanced Settings".

However, in my DBeaver Connection there is no "Advanced Settings".
![image](https://user-images.githubusercontent.com/53942137/182893480-f2de9670-3e42-4d43-9391-2e0018a30dcb.png)

So far, I have tried it with the Community Edition V22.0.0 and V22.1.3. Both are missing these "Advanced Settings"

Can you guys help me out here?

Best,
Crudelis

**Comment by LonwoLonwo** [comment]
Hello @Crudelis-rgb 

Yes, we added this setting, but we added it only in our PRO versions: https://dbeaver.com/

![2022-08-05 07_56_27-DBeaver Ultimate 22 1 4 - terms](https://user-images.githubusercontent.com/45152336/183004558-37d4c641-d507-47ba-ae11-432e72f51c87.png)

**Reply by Crudelis-rgb** [reply]
Oh ok. Thats unfortunate.. Thank you for your fast reply! :)

Any chance that this feature will make it into the community edition?
